Test correlation between module temperature and irradiance (#55)
pvlib/pvanalytics: docs/api.rst

   quality.weather.temperature_limits
   quality.weather.wind_limits

> In addition to validating temperature limits, module-temperature can
> be validated by testing whether it is correlated with irradiance. Poor

Reviewer: ```suggestion
In addition to validating temperature by comparing with limits, module 
temperature should be positively correlated with irradiance. Poor
```
